src/validate.rs

```diff
@@ -634,16 +635,14 @@ fn validate_discord_team_members_have_discord_ids(data: &Data, errors: &mut Vec<
         if team.discord_roles().is_some() && team.name() != "all" {
             let team_members = team.members(data)?;
             if team_members.len() != team.discord_ids(data)?.len() {
-                let members: String = team_members
+                let mut missing_discord_id = team_members
                     .into_iter()
-                    .filter(|name| data.person(name).map(|p| p.discord_id()) == Some(None))
-                    .map(|name| format!("{}, ", name))
-                    .collect();
+                    .filter(|name| data.person(name).map(|p| p.discord_id()) == Some(None));
```

Comment:
We can just add `.collect::<Vec<_>>();` here to avoid including `itertools` (a new dependency), I think.

Reply:
Done.